packages/graphql/src/types.ts

```diff
@@ -33,20 +31,19 @@ export type DriverConfig = {
 export interface AuthContext {
     isAuthenticated: boolean;
     roles: string[];
-    jwt?: JwtPayload;
+    jwt?: any;
 }
 
 export interface Context {
     driver: Driver;
     driverConfig?: DriverConfig;
-    jwtConfig?: Neo4jGraphQLJWT;
     resolveTree: ResolveTree;
     nodes: Node[];
     relationships: Relationship[];
     schema: GraphQLSchema;
-    jwt?: JwtPayload;
```

Comment:
This has been removed from the context here in the types, but then assigned in `wrapper.ts`?